src/main/scala/org/broadinstitute/dsde/firecloud/service/OAuthService.scala

```diff
@@ -192,7 +192,13 @@ trait OAuthService extends HttpService with PerRequestCreator with FireCloudDire
       .withPath(Uri.Path./)
       .withFragment(s"$redirectFragment?access_token=$accessToken")
 
-    redirect(Uri(uiRedirect.toString), StatusCodes.TemporaryRedirect)
+    // we require a spray DateTime, not a joda DateTime
+    // expire in one day
+    val expiration = spray.http.DateTime.now + (1000 * 60 * 60 * 24)
```

Comment:
If we don't trigger the cookie on the actual download endpoint, then I think this duration should be shortened. I think the current login expires after an hour - but I can't pin down where that is configured.